src/crab/ebpf_domain.cpp

```diff
@@ -1534,6 +1534,8 @@ void ebpf_domain_t::operator()(const Bin& bin) {
                                               dst.stack_numeric_size, src.value, false);
                                 }
                             }
+                        } else if (dst_type == T_NUM && src_type == T_NUM) {
```

Comment:
I think a comment here is in order - this case is reachable when dst and src are e.g. both {num, stack} but are unrelated, so join_over_types also enumerate the case where both are num, but they are not same_type().

Reply:
Done